discord/ext/commands/errors.py

```diff
@@ -646,7 +648,10 @@ def _get_name(x):
             try:
                 return x.__name__
             except AttributeError:
-                return x.__class__.__name__
+                pass
+            if typing.get_origin(x) is not None:
+                return repr(x)
+            return x.__class__.__name__
```

Comment:
This was added purely here so that `typing.Literal` arguments (and typing hints in general) show as their `repr` when `BadUnionArgument` is raised instead of as `_SpecialForm` or something like `_LiteralGenericAlias`, which isn't as useful to the user. How might this be moved back? (And to what `except` clause?) Or should this just be removed?

Reply:
What I meant was rather than having a `pass` within the `except` case, this code should be inside of it instead. As far as I know `typing` types do not have `__name__` so it'll fall under the `AttributeError` case.

Reply:
Oh, right, I understand now; my bad!